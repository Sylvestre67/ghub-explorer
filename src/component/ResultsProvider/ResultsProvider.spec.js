import React from 'react';
import { shallow } from 'enzyme';

import { ResultsProvider } from './index';
let component;

const props = {
	location: {
		search: 'testing',
	},
};

describe('ResultsProvider', () => {
	beforeAll(() => {
		component = shallow(<ResultsProvider {...props} />);
	});

	it('should render correctly', () => {
		expect(component).toMatchSnapshot();
	});
});
