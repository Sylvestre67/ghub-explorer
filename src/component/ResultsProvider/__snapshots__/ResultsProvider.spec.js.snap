// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsProvider should render correctly 1`] = `
<ContextProvider
  value={
    Object {
      "isFetching": true,
      "results": Object {
        "items": Array [],
      },
    }
  }
/>
`;
